### ISSUE TITLE
Dependence Problem with `wechaty-puppet-padchat@^0.8`

### ISSUE THREAD

**Issue by zixia** [lijiarui/wechaty-puppet-padchat#126]
```
11:20:34 INFO PuppetManager install(wechaty-puppet-padchat@^0.8.1) please wait ...

11:21:27 INFO PuppetManager install(wechaty-puppet-padchat@^0.8.1) done
{ Error: Cannot find module '@babel/runtime/core-js/object/create'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
```

**Comment by zixia** [comment]
This problem also exists with v0.9

This commit only fixed the pack testing, but not the actual problem: <https://github.com/lijiarui/wechaty-puppet-padchat/commit/79cdd01cf373074609be6091db6af50d3f370b86>

**Comment by windmemory** [comment]
Thanks for raise this issue :)

Fixed with ef8b40d110a7c35cf4918aa15d6c66184a4f0cd6
